nautobot/extras/models/jobs.py

```diff
@@ -359,6 +369,11 @@ def clean(self):
     def save(self, *args, **kwargs):
         """When a Job is uninstalled, auto-disable all associated JobButtons, JobHooks, and ScheduledJobs."""
         super().save(*args, **kwargs)
+        # Add specified default_job_queue to job.job_queues if it is not included.
+        if self.default_job_queue:
+            if not self.job_queues.filter(pk=self.default_job_queue.pk).exists():
+                self.job_queues.add(self.default_job_queue)
+
```

Comment:
Not sure this works as intended given the usual order of updates (`job.save()` then `job.job_queues.set(...)`) as the actual M2M update might blow this addition away. Maybe something to be done via an m2m signal handler instead?

Reply:
It seems like it is working as intended. See [test_default_job_queue_always_included_in_job_queues](https://github.com/nautobot/nautobot/blob/u/hanlin-%236259/nautobot/extras/tests/test_models.py#L1238)

Reply:
Ohhhh I see what you mean

Reply:
But would this create an "infinite loop"? Since u have to modify the m2m field within the signal handler?

Reply:
That's a good question. :) I'm not sure.